numba/cuda/libdevice.py

```diff
@@ -101,6 +112,7 @@ def core(context, builder, sig, args):
 binarys += [('__nv_copysign', '__nv_copysignf', math.copysign)]
 binarys += [('__nv_atan2', '__nv_atan2f', math.atan2)]
 binarys += [('__nv_pow', '__nv_powf', math.pow)]
+binarys += [('__nv_ldexp', '__nv_ldexpf', math.ldexp)]
```

Comment:
The second argument it integer, so adding it here would not be valid as the lowering implementation assumes floats:
https://github.com/numba/numba/pull/4855/files#diff-063ba8342a9754fcd66d73729ca1a1bdR134-R137

I expect a custom lowering function will be needed.